app/screens/navigation.ts

```diff
@@ -63,9 +63,12 @@ function registerCommandListener(name: string, params: any) {
     }
 }
 
-function screenPoppedListener({componentId}: ScreenPoppedEvent) {
+function onPoppedListener({componentId}: ScreenPoppedEvent) {
     // screen pop does not trigger registerCommandListener, but does trigger screenPoppedListener
     NavigationStore.removeScreenFromStack(componentId);
+    if (NavigationStore.getVisibleScreen() === Screens.HOME) {
+        DeviceEventEmitter.emit(Events.TAB_BAR_VISIBLE, true);
+    }
```

Comment:
not really part of this PR, but without this, the button tabs were not being displayed when returning from the channel screen.